src/product/ProductEdit.tsx

```diff
@@ -106,6 +123,54 @@ const ProductEditForm = () => {
           />
         </Box>
       </FormTab>
+
+      <FormTab label="reviews" path="reviews">
+        <ReferenceManyField
+          perPage={10}
+          reference="reviews"
+          target="product_id"
+          pagination={
+            <Pagination sx={{ alignSelf: "end", color: "text.secondary" }} />
+          }
+        >
+          <Datagrid
+            empty={
+              <Grid container justifyContent="center">
+                <Grid item>
+                  <Typography>No review yet on this poster</Typography>
+                </Grid>
+              </Grid>
+            }
+            bulkActionButtons={false}
+            sx={{
+              "& .RaDatagrid-headerCell": { fontWeight: "bold" },
+              width: "100%",
+            }}
+          >
+            <DateField source="date" />
+            <ReferenceField source="customer_id" reference="customers">
+              <CustomerField />
+            </ReferenceField>
+            <WrapperField label="Rating" sortBy="rating">
+              <Rating />
+            </WrapperField>
+            <WrapperField label="Comment">
+              <Box
+                component="div"
+                sx={{
+                  overflow: "hidden",
+                  maxWidth: "20em",
+                  whiteSpace: "nowrap",
+                  textOverflow: "ellipsis",
+                }}
+              >
+                <TextField source="comment" />
+              </Box>
+            </WrapperField>
```

Comment:
Why not just a `TextField` with the `sx` prop?

Reply:
Didn't work when I tried it. Text overflow props need to be declared in the parent node to work 🤷

Reply:
As you suggested, I created a `EllipsisTextField` component to wrap this 🙂